.github/custom-issue-template/backend-kit/1. Add linting and prettier configurations.md

```diff
@@ -2,6 +2,11 @@
 
 Having code style consistency is important for any project. A standardized set of options should be set that can be overwritten by the implementer at a later date.
 
+- ESLint: Code formatting should be enforced via ESLint rules. Rules should be set to the strictest setting and enforced via CI. The standard rules for a stack should be selected and if there are specialized plugins, those should be utilized and appropriately tuned.
```

Comment:
same comment as the FE ticket changes. i think this is too strict.

Reply:
These are taken directly from the Specification. If we want to make this less strict, we should probably visit this in the Specification.

<img width="908" alt="image" src="https://user-images.githubusercontent.com/1828602/227412656-1534ce12-e83b-4c1d-bc2d-2910206f53a3.png">